smart-table-module/js/Table.js

```diff
@@ -113,6 +113,15 @@
             };
 
             /**
+             * set lastColumnSort to defaultSortColumn if any based on column's reverse value
+             * @method setDefaultSortColumn
+             * @param column
+             */
+            function setDefaultSortColumn (defaultSortColumn) {
+                lastColumnSort = (lastColumnSort !== '') ? defaultSortColumn : lastColumnSort;
```

Comment:
Why we check `lastColumnSort` as a string and then assign a **Column** object to it?